zeebe/exporters/rdbms-exporter/src/main/java/io/camunda/exporter/rdbms/ProcessExportHandler.java

```diff
@@ -29,32 +32,39 @@ public ProcessExportHandler(final ProcessDefinitionWriter processDefinitionWrite
 
   @Override
   public boolean canExport(final Record<Process> record) {
-    return record.getIntent() == ProcessIntent.CREATED;
+    // do not react on ProcessEvent.DELETED to keep historic data
```

Comment:
💬 Should the entity be marked somehow maybe when it's deleted?

Reply:
So far neither the existing exporter does this nor does the REST Api have a field for this. But I will clarify and create a follow-up ticket if needed.